src/ecs_cluster/ecs_client.py

```diff
@@ -373,11 +373,10 @@ def ssh_to_service(self, cluster_name, service_arn, task_arn,
         else:
             ip_address = ec2_details['PrivateIpAddress']
 
-        key_name = ec2_details['KeyName']
         pem_file = self._get_ssh_key(ssh_key_dir, key_name)
 
         container_id = self._find_container_id(ip_address, task_arn)
-        docker_cmd = 'docker exec ' \
+        docker_cmd = 'sudo docker exec ' \
```

Comment:
Why do we need sudo?

Reply:
```[bhendrix@ip-10-9-0-53 ~]$ docker ps
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get http://%2Fvar%2Frun%2Fdocker.sock/v1.38/containers/json: dial unix /var/run/docker.sock: connect: permission denied
```

Reply:
You need to add your local user to the `docker` group

Reply:
Edit: nevermind, I just realized this is running on the ec2 instance, not locally.